src/test/ui/proc-macro/macro-namespace-reserved-2.stderr

```diff
@@ -93,12 +93,18 @@ error: cannot find macro `my_macro_attr` in this scope
    |
 LL |     my_macro_attr!();
    |     ^^^^^^^^^^^^^
+   |
+   = note: consider importing this attribute macro:
+           my_macro_attr
 
 error: cannot find macro `MyTrait` in this scope
   --> $DIR/macro-namespace-reserved-2.rs:33:5
    |
 LL |     MyTrait!();
    |     ^^^^^^^
+   |
+   = note: consider importing this derive macro:
+           MyTrait
```

Comment:
Hmm, this doesn't seem particularly helpful - the error is that it's in the wrong namespace, not that it's out of scope. Maybe this should say something like the mismatch for macro_rules macros?

> error: expected macro, found derive macro `crate::MyTrait`

Reply:
+1

I have spent some time trying to fix this, but unsuccessfully. So I think to emit such an error I need to compare a macro's invocation kind (`InvocationKind` in the sources) with its kind in the symbol table. My patch already prints `= note: consider importing this _derive_ macro:` (a kind of the definition), how can I get `InvocationKind`?